gems/smithy/lib/smithy/templates/client/protocol_spec.erb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+# This is generated code!
+
+require_relative '../spec_helper'
+<% additional_requires.each do |r| -%>
+require '<%= r %>'
+<% end -%>
+
+
+module <%= module_name %>
+  # TODO: Can be replaced by stub_responses config once implemented
+  class StubSend < Smithy::Client::Plugin
```

Comment:
You could reuse the dummy send handler in spec helper couldn't you?

Reply:
What dummy send handler - I couldn't find it? It also has the issue that it would need to be available as a test dependency for generated gems.  Code generating a small plugin as a stop gap until we have stup_responses here seems easier than trying to take more test dependencies.

Reply:
I think Matt was referring to this: https://github.com/smithy-lang/smithy-ruby/blob/decaf/gems/smithy-client/spec/spec_helper.rb#L15

I think this is fine long as there's TODO

Reply:
Ah I see, was looking in Smithy.  

Yeah, the same issue applies - we don't want to move that dummy send plugin to the lib directory in smithy-client so that it can be used as a test dependency in generated gems.